### PR TITLE
Add line sso

### PR DESCRIPTION
**Pull request by jimmyyyeh**
Integration with Line [ref.](https://developers.line.biz/en/docs/line-login/integrate-line-login/)

### CODE REVIEW

File: fastapi_sso/sso/line.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            'token_endpoint': f'{self.base_url}/token',
+            'userinfo_endpoint': f'{self.base_url}/userinfo'
+        }
+
+    async def openid_from_response(self, response: dict, session: Optional['httpx.AsyncClient'] = None) -> OpenID:
+        '''Return OpenID from user information provided by Line'''
+        return OpenID(
+            email=response.get('email'),
+            first_name='',
+            last_name='',
```
Comment (tomasvotava) [inline fastapi_sso/sso/line.py:33]: both `first_name` and `last_name` are optional on `OpenID`, I'd prefer to just omit them

**Review by tomasvotava — CHANGES_REQUESTED** [review]
Found only a minor styling issue - I think the pipeline will fail, all other files are using double quotes `"`, could you please try running `poe black` locally as pointed out in the contributing guide? https://tomasvotava.github.io/fastapi-sso/contributing/

Also, please add:
- example to `examples/` subdir
- your handle and name (if you want to) to `README.md`

If you have any problems, let me know, I'll add those for you.

Thanks a lot for the contribution!

File: fastapi_sso/sso/line.py (hunk 1 above)
Comment (tomasvotava) [reply fastapi_sso/sso/line.py]: Also, the linter found some unused imports and stuff, take a look at the pylint job or run `poe lint` locally to see and try to tackle the issues. If you have any troubles, let me know 👍

Comment (jimmyyyeh) [reply fastapi_sso/sso/line.py]: Line didn't provide those information, so I think is better return empty string.

Comment (tomasvotava) [reply fastapi_sso/sso/line.py]: Both default to `None`, I feel it's more convenient to check for nones than empty strings, will I change your mind? :smile: https://github.com/tomasvotava/fastapi-sso/blob/master/fastapi_sso/sso/base.py#L49

Comment (jimmyyyeh) [reply fastapi_sso/sso/line.py]: sure, I'll change it to None.

**Review by tomasvotava — APPROVED** [review]

**Pull request merged**